packages/rancher-istio/charts/templates/admin-role.yaml

```
@@ -4,7 +4,7 @@ kind: ClusterRole
 metadata:
   labels:
     rbac.authorization.k8s.io/aggregate-to-admin: "true"
-  namespace: istio-system
+  namespace: {{ .Values.global.istioNamespace | default "istio-system" }}
```

Should this be changed from `istio-system` to `cattle-istio-system`?

```suggestion
  namespace: {{ .Values.global.istioNamespace | default "cattle-istio-system" }}
```

---

by default istio deploys to istio-system if the istioNamespace value is not provided, so the roles and services accounts need to do the same

---

This may not be consistent with the default set here right to fall back to `cattle-istio-system`? https://github.com/rancher/charts/pull/531#discussion_r468087217.